dev-db/sqlitebrowser/sqlitebrowser-3.9.1.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="DB Browser for SQLite is a high quality, visual, open source tool to create, design, and edit database files compatible with SQLite."
+DESCRIPTION="It is for users and developers wanting to create databases, search, and edit data. It uses a familiar spreadsheet-like interface, and you don't need to learn complicated SQL commands.\
+Controls and wizards are available for users to:\
+Create and compact database files\
+Create, define, modify and delete tables\
+Create, define and delete indexes\
+Browse, edit, add and delete records\
+Search records\
+Import and export records as text\
+Import and export tables from/to CSV files\
+Import and export databases from/to SQL dump files\
+Issue SQL queries and inspect the results\
+Examine a log of all SQL commands issued by the application"
+HOMEPAGE="https://github.com/sqlitebrowser/sqlitebrowser"
+COPYRIGHT="Copyright notice>"
+LICENSE="GNU GPL v3"
+REVISION="1"
+scrGitRev="70ff691f1a8a82c91c22160c6d990fbee9bc0eb7"
+SOURCE_URI="$HOMEPAGE/archive/$scrGitRev.tar.gz"
+CHECKSUM_SHA256=""
+SOURCE_DIR="sqlitebrowser-$scrGitRev"
```

Comment:
@dash102, yes, we only use "srcGitRev=" (or isometimes "commit=", which is equivalent) for programs which are missing a usable stable release.
Since 3.9.1 has an official tarball, we can use it.
(You can copy/paste what I wrote in my first review. I have even computed the sha256 checksum for you ;-)
BTW, regarding the checksum, I should have checked that I did get a copy of a genuine tarball, but I did not. I sometimes compare what I get with what other distros have in their packages. That's not a good practice, I know... Verifying digital signature is important to make sure we are not building _modified_ sources.